temporal-sdk/src/test/java/io/temporal/workflow/shared/TestWorkflows.java

```diff
@@ -89,6 +90,12 @@
     String execute(String arg, int arg2);
   }
 
+  @WorkflowInterface
+  public interface TestWorkflow4 {
```

Comment:
Maybe `TestWorkflowReturnBoolean` to keep things consistent with `TestWorkflowReturnString` and to simplify reusing?